book/src/user/mining.md

```diff
@@ -0,0 +1,167 @@
+# Mining Zcash with zebra
+
+Zebra's RPC methods support miners and mining pools. 
+
+Contents:
+
+- [Download and build Zebra](#download-and-build-zebra)
+- [Configure zebra for mining](#configure-zebra-for-mining)
+  - [Miner address](#miner-address)
+  - [RPC section](#rpc-section)
+- [Running zebra](#running-zebra)
+- [Testing the setup](#testing-the-setup)
+- [Run a mining pool](#run-a-mining-pool)
+
+## Download and build Zebra for mining
+[#download-and-build-zebra]: #download-and-build-zebra
+
+```console
+cargo install --locked --features getblocktemplate-rpcs --git https://github.com/ZcashFoundation/zebra zebrad
+```
+
+Zebra binary will be at ` ~/.cargo/bin/zebrad`.
+
+It is important to install with the `getblocktemplate-rpcs` feature so the final binary will contain mining support.
+
+## Configure zebra for mining
+[#configure-zebra-for-mining]: #configure-zebra-for-mining
+
+We need a configuration file. First, we create a file with the default settings:
+
+```console
+zebrad generate -o ~/.config/zebrad.toml
+```
+
+The above command places the generated `zebrad.toml` config file in the default preferences directory of Linux. For other OSes default locations [see here](https://docs.rs/dirs/latest/dirs/fn.preference_dir.html).
+
+Tweak the following options in order to prepare for mining.
+
+### Miner address
+[#miner-address]: #miner-address
+
+Node miner address is required. At the moment zebra only allows `p2pkh` or `p2sh` transparent addresses.
+
+```
+[mining]
+miner_address = 't3K4aLYagSSBySdrfAGGeUd5H9z5Qvz88t2'
```

Comment:
Where is this address from? (Which network and who has the keys?)

This needs to go as a note in the docs, so we are transparent about who gets the funds from it.